partners/scientific-communities.md

```diff
@@ -1,38 +1,47 @@
-# Partnerships with Scientific Python Communities
+# Partnerships with scientific Python Communities
 
-```{toctree}
+
+
+:::{toctree}
 :hidden:
 :caption: Our Partners
 
 About Our Partnerships <self>
 JOSS <joss>
 Pangeo <pangeo>
-```
+:::
+
 
-Numerous scientific Python communities are developing their own peer review
-processes for affiliated packages. Communities with their own package
-review processes can leverage our existing peer review process, which is
-built on community-driven package standards. In partnership with these
-communities, we enhance our standard review process with a community-specific
+:::{figure-md} pyopensci-3-pillars
+
+<img src="../images/pyopensci-pillars-flower.png" alt="Flower" width="500">
+
+pyOpenSci has three core programs. Community partnerships is a program that allows domain specific scientific Python communities to leverage our peer review process rather than creating their own process. Through this partnership maintainers can become affiliated with both a domain specific community, published in the Journal of Open Source Software and also become a part if the pyOpenSci community.
+:::
+
+Numerous science domain-specific scientific Python communities are developing and maintaining their own peer review
+processes to track and recommend packages that relevant to their science domain.
+However, running a high quality peer review process takes time and resources.
+
+pyOpenSci offers a program where communities can partner with us, and leverage our peer review process, which is
+built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a community-specific
```

Comment:
```suggestion
built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a custom review process
```

Reply:
i really appreciate the concept here but i'm slightly worried about the word "custom" which might make it sound like we can totally modify our process. if there is a better word to suggest adding a layer on top of an existing structure i think that would be better. Community specific related to adding guidelines specific to that community but the process is still the same (with that additional layer). i know this is very nuanced but i wouldn't want someone to think we can totally customize our review process as that would be too much work to maintain. i'll think about other words or perhaps you have ideas too!! i may merge this before you are back only because of astropy moving forward now BUT  we can also always go back in and modify!

Reply:
ok i know where this is from now. and i'll think about what words we want to use. it's a custom step rather than a custom review process. maybe i'll adjust the graphic language and finesse this...